about.md

```diff
@@ -18,23 +18,54 @@ They bring together design, technical, and policy components to give a view of w
 
 Working with users to develop trusted data patterns can also help us avoid dark patterns.
 
-[Dark patterns](https://www.darkpatterns.org/) are "tricks used in websites and apps that make you do things that you didn't mean to, like buying or signing up for something". They [use the power of design](https://dapde.de/en/) to benefit a product's owners or creators more than the people who use it.
+[Dark patterns](https://www.darkpatterns.org/) are "tricks used in websites and apps that make you do things that you didn't mean to, like buying or signing up for something".
+They [use the power of design](https://dapde.de/en/) to benefit a product's owners or creators more than the people who use it.
 
 Dark patterns can do this in many different ways, but mainly by limiting:
 
 * **Consent** - the ability for people to choose what data they do and don't want to have collected, used, stored or shared
-* **Openness** - the ability for people to see and understand how a service was designed and how it works
+* **Openness** - the ability for people to see, understand, and participate in how a service is designed and how it works
 * **Accountability** - the ability to give, receive, and respond to feedback
 
-For example,
+#### Example - Cookie pop-up
+
+For example, we're required by law to ask people for their consent when we want to collect data through cookies.
+This is often achieved by asking people to consent via a cookie pop-up.
+
+A dark pattern for a cookie pop-up might lead you to share more data than you are comfortable with by assuming you want to consent to all cookies, then making it difficult to change your preferences.
+
+One way to do this is through deceptive patterns like **misdirection** - where a visual element like a button is used to distract from content and lead the user to take an action they might not otherwise.
+
+<img width="600" height="300" src=https://user-images.githubusercontent.com/47132182/127548057-25e2a63d-cd39-44e4-93e4-26cf1748df8a.png>
+
+Another way is through a combination of obstructive patterns including:
+
+* **Pre-selection** - where settings options are selected by default
+* **Click fatigue** - where a user is led to accept a default or easy path by making click paths of different lengths/effort
+* **Hidden information** - where useful options or details are hidden or difficult to access
+
+<img width="600" height="300" src="https://user-images.githubusercontent.com/47132182/127549679-6403d448-1fb0-4d3f-afb9-b956be99dc86.png">
+
+Our data patterns aim to counteract dark patterns by supporting and expanding consent, openness, and accountability rather than restricting them.
+
+A few ways we can do this include:
+
+* Making information **accessible**, with **varying levels of detail**
+* **Not pre-selecting options** or making default assumptions, unless validated by user needs
+* Providing one or low click options to **change settings in bulk** _and_ options to **change individual settings**
```

Comment:
We'll have to talk about what the right way to show this pattern is, because there are some issues with it (I'm making assumptions based on the image)

- The label can't perform 2 actions, it has to only select or de-select the field itself, so it can't also act to expand for more details (we could have the short text under it be the clickable thing to expand)
- I believe the "select all", and "clear all" need to be buttons and need to be both at the top and bottom of the list.
- I'm also pretty sure the submit and the "checkbox"(or slider) needs to be to the left side of the "page" to enable smoother consistent eye flow

Of course in the moment I can't find the documentation to back up these claims 🙃 
We make make these changes later, so this comment is non-blocking.